scrapy/pipelines/images.py

```diff
@@ -88,6 +88,8 @@ def from_settings(cls, settings):
         s3store = cls.STORE_SCHEMES['s3']
         s3store.AWS_ACCESS_KEY_ID = settings['AWS_ACCESS_KEY_ID']
         s3store.AWS_SECRET_ACCESS_KEY = settings['AWS_SECRET_ACCESS_KEY']
+        s3store.POLICY = settings['FILES_STORE_S3_ACL']
```

Comment:
I'd go for moving `FILES_STORE_S3_ACL` to `docs/topics/media-pipeline.rst`, and document `IMAGES_STORE_S3_ACL` there too.